### PR TITLE
OCPBUGS-33953: Document how ICSP/IDMS work in HCP

### PR DESCRIPTION
**Pull request by jparrill**
**Which issue(s) this PR fixes** :
Fixes #[OCPBUGS-33953](https://issues.redhat.com/browse/OCPBUGS-33953)

This PR depends on #4086, once this got merged this one will only contain one commit.

### CODE REVIEW

**Comment by enxebre** [comment]
/hold
goes after https://github.com/openshift/hypershift/pull/4086

File: docs/content/how-to/disconnected/known-issues.md [hunk 1]
```diff
@@ -189,3 +189,56 @@ for isName in ${IMAGESTREAMS}; do
...
+
+**Symptoms:**
+Even having the ICSP/IDMS well set in the Management cluster and working fine in that side, the HostedControlPlane deployment is not capable of extract the metadata release fromthe OCP payload images. Something like this log entry will show up in the ControlPlaneOperator or HypershiftOperator pod.
+
+```
+failed to lookup release image: failed to extract release metadata: failed to get repo setup: failed to create repository client for https://registry.sample.net: Get "https://registry.sample.net/v2/": tls: failed to verify certificate: x509: certificate signed by unknown authority
+```
+
+**Root Cause**:
+Once the ICSP/IDMS are created in the Management cluster and they are only using in the "sources" side a root registry instead of pointing a registry namespace, the image-overrides are not properly propagated through all the deployments in the HostedControlPlane namespace. This is how looks like:
```
Comment (enxebre) [inline docs/content/how-to/disconnected/known-issues.md:205]: Is this saying the API input is legit but we don't propagate it properly? can we fix that?

Comment (jparrill) [reply docs/content/how-to/disconnected/known-issues.md:205]: Yes, I have a bug opened but it's set to the next sprint so maybe will not get in to 4.16.0 relase cut

Comment (enxebre) [reply docs/content/how-to/disconnected/known-issues.md:205]: can you please share link?

Comment (jparrill) [reply docs/content/how-to/disconnected/known-issues.md:205]: https://issues.redhat.com/browse/OCPBUGS-33951

**Pull request merged**